packages/mdk4/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_HOMEPAGE=https://github.com/aircrack-ng/mdk4
+TERMUX_PKG_DESCRIPTION="MDK is a proof-of-concept tool to exploit common IEEE 802.11 protocol weaknesses."
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_VERSION=4.1
+TERMUX_PKG_REVISION=
```

Comment:
Remove this line completely, no need to set it

Reply:
removed done.